lib/surface/live_component.ex

```diff
@@ -65,20 +65,16 @@ defmodule Surface.LiveComponent do
       @behaviour unquote(__MODULE__)
       require Surface.ContentHandler
       @before_compile Surface.ContentHandler
-      Module.put_attribute(__MODULE__, :__is_stateful__, true)
```

Comment:
This was added in the compiler, but ended up not being used - instead we just use the `Surface.LiveComponent`/`Surface.Component` distinction to determine if it's stateful or not